starter-code/public/scripts/controllers/articleController.js

```diff
@@ -3,10 +3,14 @@ var app = app || {};
 
 (function(module) {
   const articleController = {};
-  // TODO: Setup a function that kicks off the fetching and rendering of articles, using the same
+  // Done!TODO: Setup a function that kicks off the fetching and rendering of articles, using the same
   // code that used to be in index.html.
   // Also be sure to hide all the main section elements, and reveal the #articles section:
-
+  articleController.init = function () {
+    $('.tab-content').hide();
+    $('#articles').show();
+    app.Article.fetchAll(module.articleView.initIndexPage);
+  }
```

Comment:
Great! The only comment I would make on this is to use either app or module for both your fetchAll and your callback